op-monitorism/faultproof_withdrawals/cli.go

```diff
@@ -0,0 +1,99 @@
+package faultproof_withdrawals
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	L1GethURLFlagName = "l1.geth.url"
+	L2NodeURLFlagName = "l2.node.url"
+	L2GethURLFlagName = "l2.geth.url"
+
+	EventBlockRangeFlagName           = "event.block.range"
+	StartingL1BlockHeightFlagName     = "start.block.height"
+	HoursInThePastToStartFromFlagName = "start.block.hours.ago"
+
+	OptimismPortalAddressFlagName = "optimismportal.address"
+)
+
+type CLIConfig struct {
+	L1GethURL   string
+	L2OpGethURL string
+	L2OpNodeURL string
+
+	EventBlockRange           uint64
+	StartingL1BlockHeight     int64
```

Comment:
why this one is not uint64?
This can be negative value? if yes why?

Reply:
Because we need a default value to test if it was set or not in the parameters.